src/NUnitEngine/nunit.engine/Runners/ProcessRunner.cs

```diff
@@ -31,10 +31,18 @@
 namespace NUnit.Engine.Runners
 {
     /// <summary>
-    /// Summary description for ProcessRunner.
+    /// ProcessRunner loads and runs a set of tests in a single age process.
```

Comment:
"Single age process"

Is this a typo, or just terminology I've never come across before? 😅 Google couldn't help me with the terminology, but I can't work out what it could be a typo for either!

Reply:
"agent"

Reply:
Ah! That one didnt cross my mind! I can correct it now, thanks!